packages/application/src/requests.tsx

```diff
@@ -82,7 +109,18 @@ export class RequestHandler {
       displayWaitDialog
     );
   }
-
+  /**
+   * Make an request to the jupyterlab server.
+   *
+   * @param requestExt - The url for the request.
+   *
+   * @param requestOptions - The initialization options for the request.
+   *
```

Comment:
What are these options? Valid values?

Reply:
this is passed as is to `makeRequest` but that function calls it `init` and defines it as `The initialization options for the request` should we change our wording to match?

Reply:
As a consumer, how do I figure out what valid options that I should use here? Is this based on some other function parameter field requirement/definition? is there any documentation available to help consumers?